src/templates/prototypes/constituencies.pug

```diff
@@ -1,16 +1,18 @@
 extends ../../layouts/layout.pug
 block content
-	-var thingPage
+  -var thingPage
 
-	+section--primary
-		//- Page Title
-		h1 Current and former constituencies
+  .section--primary#content
```

Comment:
If not a thingPage then content ID needs to go the next section, so that it skips the navigation--letter.